### PR TITLE
refactor redhat tasks order

### PR DESCRIPTION
**Pull request by lhoss**
...that should not break anything ( also reordering tasks will not influence the 'restart handlers' , that are only called at the end of the playbook)

Motivation: https://github.com/AnsibleShipyard/ansible-zookeeper/issues/26#issuecomment-237172988

I will create separate PRs for the different refactoring steps as requested by @ernestas-poskus
(ps: next refactor PR will follow within 1 hour.. hopefully we can get those merged quickly, else I will have to create a chain of branches/PRs depending on each other)

### CODE REVIEW

File: tasks/RedHat.yml [hunk 1]
```diff
@@ -1,15 +1,17 @@
 ---
+- name: Install OS Packages
+  yum: name={{ item }} state=present
+  with_items:
+    - libselinux-python
+
+
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: extra empty line

File: tasks/RedHat.yml [hunk 2]
```diff
@@ -30,6 +32,21 @@
   file: path={{log_dir}} state=directory recurse=yes owner=zookeeper group=zookeeper
   tags: bootstrap
 
+
+- name: Write myid file.
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: extra empty line

File: tasks/RedHat.yml [hunk 3]
```diff
@@ -30,6 +32,21 @@
   file: path={{log_dir}} state=directory recurse=yes owner=zookeeper group=zookeeper
   tags: bootstrap
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: extra empty line

File: tasks/RedHat.yml [hunk 4]
```diff
@@ -30,6 +32,21 @@
...
+  tags: deploy
+  notify:
+    - Restart zookeeper
+
+- name: Configure zookeeper
+  template: src=zoo.cfg.j2 dest={{ zookeeper_dir }}/conf/zoo.cfg owner=zookeeper group=zookeeper
+  tags: deploy
+  notify:
+    - Restart zookeeper
+
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: no need for extra empty line

File: tasks/RedHat.yml [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
+
 - file: path=/opt/src state=directory
 - file: path={{zookeeper_dir}} state=directory
 
 - name: Download zookeeper version.
   get_url: url={{zookeeper_url}} dest=/opt/src/zookeeper-{{zookeeper_version}}.tar.gz
   tags: bootstrap
 
-- name: Install OS Packages
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: why you have moved this ?

File: tasks/RedHat.yml [hunk 6]
```diff
@@ -41,6 +58,7 @@
   notify:
     - Restart zookeeper
 
+
```
Comment (ernestas-poskus) [inline tasks/RedHat.yml]: no need for extra empty line

**Comment by ernestas-poskus** [comment]
please remove extra lines

**Comment by lhoss** [comment]
@ernestas-poskus thx for review!
Removing the empty lines here wouldn't make much sense (except create intermediary commits and work) because in the succeeding PR (ready for review), they might have been removed already (if not, I'm happy to remove them there, no later than tomorrow)

**Comment by ernestas-poskus** [comment]
remove extra empty lines and I will merge it

**Comment by lhoss** [comment]
closed in favor of https://github.com/AnsibleShipyard/ansible-zookeeper/issues/33

**Pull request closed without merging**